src/components/TabPanel/TabPanel.tsx

```diff
@@ -15,31 +12,39 @@ interface TabPanelProps {
   isHidden: boolean;
   errors?: YupError[];
   onClick: (event: React.MouseEvent<any>) => void;
+  heading: string;
 }
 
+const StyledHeading = styled.div`
+  font-size: 2rem;
+  font-weight: bold;
+  line-height: 1.5rem;
+  margin-top: 2rem;
+  margin-bottom: 1rem;
+`;
+
 const StyledButton = styled(Button)`
   margin-top: 1rem;
   margin-right: 0.5rem;
 `;
 
-const TabPanel: React.FC<TabPanelProps> = ({ ariaLabel, children, errors, isHidden, onClick }) => {
+const TabPanel: React.FC<TabPanelProps> = ({ ariaLabel, children, errors, isHidden, onClick, heading }) => {
   const { t } = useTranslation();
 
   return (
     <Typography component="div" role="tabpanel" hidden={isHidden} aria-labelledby={`nav-tab-${ariaLabel}`}>
-      <Box>
-        {errors &&
-          errors.length > 0 &&
-          errors.map((error: any) => {
-            return <LabelTextLine key={error.path} label={error.path} text={`${error.name} - ${error.message}`} />;
-          })}
-        {children}
-        {onClick && (
-          <StyledButton color="primary" variant="contained" onClick={onClick}>
-            {t('Next')}
-          </StyledButton>
-        )}
-      </Box>
+      {errors &&
+        errors.length > 0 &&
+        errors.map((error: any) => {
+          return <LabelTextLine key={error.path} label={error.path} text={`${error.name} - ${error.message}`} />;
+        })}
+      <StyledHeading>{t(heading)}</StyledHeading>
```

Comment:
Er det ikke mest naturlig at `heading` allerede er oversatt? Altså at det håndteres av parent

Reply:
Har ikke så mye å si siden vi uansett må bruke `t` for "Next" på linje 45, så whatever

Reply:
burde hatt en standard for dette. kan diskuteres ved en annen anledning!